### ISSUE TITLE
Settings / Database / Select Directory dialog

### ISSUE THREAD

**Issue by jgaa** [jgaa/f-crm#8]
The dialog is confusing, as it says "Save" while in fact it will open the selected database the next time the application is opened.

**Comment by nafergo** [comment]
True. After changing the database path you need to reopen to use it. In the beginning I thought that you just needed to change the path so that f-crm would automagically start using it :) like when I change client info (for example) - just need to change and f-crm autosaves :)

**Comment by jgaa** [comment]
I'll see if I can change this, so that it's more intuitive.
